secondary-navigation.yaml

```diff
@@ -84,7 +84,7 @@ lxd:
     - title: Manage
       path: /lxd/manage
     - title: Docs
-      path: https://documentation.ubuntu.com/lxd/en/latest/?_ga=2.91941594.1329653141.1700868845-1063360542.1700868845
+      path: https://documentation.ubuntu.com/lxd/en/stable-5.21
```

Comment:
The link in the doc is https://canonical-lxd.readthedocs-hosted.com/en/, which then redirects to the above link. Is this intentional?